eng/formatting/format.sh

```diff
@@ -17,7 +17,7 @@ fi
 
 if [ -n "$MANAGED_FILES" ]; then
     # Format all selected files
-    echo "$MANAGED_FILES" | cat | xargs | sed -e 's/ /,/g' | dotnet format --no-restore --include -
+    echo "$MANAGED_FILES" | cat | xargs | sed -e 's/ /,/g' | dotnet format whitespace --include - --folder
```

Comment:
Is this script invoked in CI or just a dev innerloop thing?

Reply:
Just a dev innerloop thing.